command/fs/locator.go

```diff
@@ -238,15 +238,21 @@ func (f *Locator) loadDirContents() {
 		return
 	}
 	for _, finfo := range contents {
-		if f.mod == Files && finfo.IsDir() {
-			continue
-		} else if f.mod == Dirs && !finfo.IsDir() {
-			continue
+		if f.mod.match(finfo) {
+			name := finfo.Name()
+			if finfo.IsDir() {
+				name += string(filepath.Separator)
+			}
+			f.files = append(f.files, name)
 		}
-		name := finfo.Name()
-		if finfo.IsDir() {
-			name += string(filepath.Separator)
-		}
-		f.files = append(f.files, name)
 	}
 }
+
+func (m Mod) match(finfo os.FileInfo) bool {
+	if m == Files {
+		return !finfo.IsDir()
+	} else if m == Dirs {
```

Comment:
This `else` isn't necessary, and I think this works better as a `switch` statement.

I also just generally don't ever write code that uses `else` statements - [`Effective Go`](https://golang.org/doc/effective_go.html#control-structures) explains some of why.  The short version is that else statements cause readability problems, forcing us to think in branches of logic that are a heavy load on our memory.  Code without else statements is much easier to reason about.